packages/lodestar-spec-test-util/src/downloadTests.ts

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+import tar from "tar";
+import stream from "stream";
+import {promisify} from "util";
+
+const BASE_URL = "https://github.com/ethereum/eth2.0-spec-tests/releases/download";
+
+// eslint-disable-next-line prettier/prettier
+const TESTS = ["general", "mainnet", "minimal"];
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+export async function downloadTests(
+  {specVersion, outputDir}: {specVersion: string; outputDir: string},
+  log?: (msg: string) => void
+): Promise<void> {
+  if (log) log(`outputDir = ${outputDir}`);
+
+  await Promise.all(
+    TESTS.map(async (test) => {
+      const URL = `${BASE_URL}/${specVersion}/${test}.tar.gz`;
+      const OUTPUT = `${outputDir}`;
+
+      // download tar
+      const {data, headers} = await axios({
```

Comment:
can we check if tests are already downloaded and extracted?

Reply:
What condition should it use for that? That there are some files in `outputDir`? That there are files with certain names in `outputDir`?

Reply:
We can check if we have {specVersion} directory and not download then

Reply:
Would prefer to have this logic outside this function/file. "if tests don't exist, run this function"
We may have this condition checked in a github action, for example. "if cache hit, restore tests, else, download tests"

Reply:
I would keep it in script if not in this method.